dev/integration_tests/flutter_gallery/lib/gallery/themes.dart

```diff
@@ -30,7 +30,6 @@ ThemeData _buildDarkTheme() {
     primaryColorDark: const Color(0xFF0050a0),
     primaryColorLight: secondaryColor,
     indicatorColor: Colors.white,
-    toggleableActiveColor: const Color(0xFF6997DF),
```

Comment:
Just a note for follow-up, this is a copy of flutter gallery. The real version in flutter/gallery will need this change as well.

https://github.com/flutter/gallery/blob/637e6a327571da6c3c4011dfa509ecc4377c1ad4/lib/themes/material_demo_theme_data.dart#L19

Reply:
That makes sense, i will create PR for that once tests are passing.